librz/core/cmd/cmd_eval.c

```diff
@@ -53,8 +53,39 @@ static bool pal_seek(RzCore *core, RzConsPalSeekMode mode, const char *file, RzL
 	return true;
 }
 
-RZ_API bool rz_core_theme_load(RzCore *core, const char *name) {
+static bool print_meta_info(RzCore *core, RzCmdStateOutput *state, RzThemeInfo *meta_info) {
 	bool failed = false;
+
+	if (meta_info->author != NULL) {
+		rz_cons_printf("%s", meta_info->author);
+	}
+	if (meta_info->license != NULL) {
+		rz_cons_printf("%s", meta_info->license);
+	}
+	if (meta_info->targetbg != NULL) {
+		rz_cons_printf("%s", meta_info->targetbg);
+	}
+	if (meta_info->mincolordepth != NULL) {
+		rz_cons_printf("%s", meta_info->mincolordepth);
+	}
+	if (meta_info->desc != NULL) {
+		rz_cons_printf("%s", meta_info->desc);
+	}
+	if (meta_info->additionaldts != NULL) {
+		rz_cons_printf("%s", meta_info->additionaldts);
+	}
+	if (meta_info->refs != NULL) {
+		rz_cons_printf("%s", meta_info->refs);
+	}
+
+	return (!failed);
+}
+
+RZ_API bool rz_core_theme_load(RzCore *core, const char *name, RzCmdStateOutput *state) {
```

Comment:
this function should just "load", as the name implies. this function should at most load the metainfo in the theme structure, but not print them IMHO.

Reply:
Completely agree with this! Ive got an idea to refactor the functionalities, will push a commit soon.